scripts/lib.py

```diff
@@ -96,7 +96,7 @@ def get_seed(seed):
   """
   if seed >= 0:
     return seed
-  return random.getrandbits(32)
+  return random.getrandbits(31)
```

Comment:
returning unsigned value of 32 bits is out of range for 32 bits signed seed
function was changed to return 31 bits